__tests__/api/topics/follow-v2.test.js

```diff
@@ -21,10 +21,10 @@ describe('PUT /topics/follow-v2', () => {
       .send({
         name: 'Topic 1'
       });
-
     // Assertion
     expect(response.statusCode).toBe(200);
-    expect(response.body.message).toBe('Success delete topic user v2');
+    expect(response.body.data[0].message).toBe('Success delete topic user v2');
+    expect(response.body.data[0].type).toBe('unfollow');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The assertions have been updated to check the `message` and `type` fields within the first object of the `data` array in the response body. This change is consistent with the PR summary, which mentions that the response data now includes an array of objects detailing the follow/unfollow action. However, it's important to ensure that the response always contains at least one object in the `data` array before accessing its properties to avoid potential runtime errors.
```diff
- expect(response.body.data[0].message).toBe('Success delete topic user v2');
- expect(response.body.data[0].type).toBe('unfollow');
+ expect(response.body.data.length).toBeGreaterThan(0);
+ if (response.body.data.length > 0) {
+   expect(response.body.data[0].message).toBe('Success delete topic user v2');
+   expect(response.body.data[0].type).toBe('unfollow');
+ }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->